libflux/flux-core/src/formatter/tests.rs

```diff
@@ -1268,3 +1268,17 @@ a",
     );
     assert_unchanged(r#"a = "a string literal with a tabstop '	'""#);
 }
+
+#[test]
+fn invalid_syntax() {
+    let script = r#"builtin diff : (
+        <-got: [A],
+        want: [A],
+        ?verbose: bool,
+        ?epsilon: float,
+        ?nansEqual: bool,
+        aoeustahoesih
```

Comment:
Haha! I know why this random string is here, but it's going to result is some really fun lore in the future. 😄